### PR TITLE
docs: quickstart iOS swift

### PR DESCRIPTION
**Pull request by sixhobbits**
depends on https://github.com/ritza-co/fusionauth-quickstart-swift-ios-native being moved to fusionauth org

### CODE REVIEW

File: astro/src/content/quickstarts/quickstart-swift-ios-native.mdx [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+
+Select <Breadcrumb>File -> Add Package Dependency</Breadcrumb> from the menu bar. In the search bar, paste in `https://github.com/openid/AppAuth-ios`. In the list of packages select "appauth-ios" and click <Uielement label="Add Package"/>. You will be asked to choose the packages to install. On the "Add to Target" column select `Changebank` for both "AppAuth" and "AppAuthCore". Click <Uielement label="Add Package"/> to add the packages to your project.
+
+## Customization
+
+Before you start coding, you need to add some assets and styles to your project that will be referenced later.
+
+### Add Assets And Styles
+
+Add the ChangeBank logo to the project. Download it from [here](https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-swift-ios-native/main/complete-application/ChangeBank/Assets.xcassets/changebank.imageset/changebank.png). Click on the <Uielement label="Assets.xcassets"/> directory in the project navigator to open the asset manager. Drag and drop the downloaded `ChangeBank.png` file into the asset manager.
```
Comment (ColinFrick) [inline astro/src/content/quickstarts/quickstart-swift-ios-native.mdx:117]: Nitpick, but Assets.xcassets is just Assets in my XCode:
ChangeBank

Comment (ColinFrick) [reply astro/src/content/quickstarts/quickstart-swift-ios-native.mdx:117]: Additionally, file is all lower case `changebank.png`, and not PascalCase.

File: astro/src/content/quickstarts/quickstart-swift-ios-native.mdx [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+
+### Running The App On A Device
+
+To run the app on your device, you need to make a few changes. This is because the app will be running on a different device to the FusionAuth instance, so it won't be able to access the FusionAuth instance via `localhost`.
+
+A simple way to get around this is to use [ngrok](https://ngrok.com/), a tool that allows you to expose a local server to the internet. You can follow the [guide here](/docs/get-started/download-and-install/development/exposing-instance). Note the URL ngrok gave you as you'll need it soon.
+
+Now that you have the URL, you need to update the FusionAuth Tenant issuer to make sure it matches the given address.
+
+Log in to the [FusionAuth admin UI](http://localhost:9011/admin), browse to <Breadcrumb>Tenants</Breadcrumb> in the sidebar, and click <IconButton icon="edit" color="blue" /> on the "Default" tenant to edit it. Paste the complete address _(with protocol and domain)_ you copied from ngrok into the <Field label="Issuer"/> field (for example, `https://6d1e-2804-431-c7c9-739-4703-98a7-4b81-5ba6.ngrok-free.app`). Save the application by clicking the <IconButton icon="save" color="blue" /> icon in the top-right corner.
```
Comment (ColinFrick) [inline astro/src/content/quickstarts/quickstart-swift-ios-native.mdx:246]:
```suggestion
Log in to the [FusionAuth admin UI](http://localhost:9011/admin), browse to <Breadcrumb>Tenants</Breadcrumb> in the sidebar, and click <IconButton icon="edit" color="blue" /> on the "Default" tenant to edit it. Paste the complete address _(with protocol and domain)_ you copied from ngrok into the <Field label="Issuer"/> field (for example, `https://6d1e-2804-431-c7c9-739-4703-98a7-4b81-5ba6.ngrok-free.app`). Save the tenant by clicking the <IconButton icon="save" color="blue" /> icon in the top-right corner.
```

**Review by ColinFrick — CHANGES_REQUESTED** [review]
The `kickstart/styles.css` uses a blue color for `--main-accent-color` and not the ChangeBank green. For example https://github.com/FusionAuth/fusionauth-quickstart-python-flask-web/blob/main/kickstart/css/styles.css

When iOS is configured to use Dark mode, the ChangeBank logo is not displayed correctly:
image

Maybe force the App to use Light mode?

File: astro/src/content/quickstarts/quickstart-swift-ios-native.mdx (hunk 1 above)
Comment (rideam) [reply astro/src/content/quickstarts/quickstart-swift-ios-native.mdx]: Your preference is set to hide the file extensions thats why it is only showing the file name `Assets`, updated to only reflect the name

**Review by ColinFrick — APPROVED** [review]
LGTM

**Comment by mooreds** [comment]
Closing under https://github.com/FusionAuth/fusionauth-site/pull/2737 because of one small change. But looks great!

**Pull request closed without merging**